tcms/rpc/tests/test_testcase.py

```diff
@@ -138,6 +138,42 @@ def test_update_priority_issue_1318(self):
         self.assertEqual('Sanity test case', self.testcase.summary)
         self.assertEqual('Given-When-Then', self.testcase.text)
 
+    def test_update_default_tester_accepts_user_id(self):
+        self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+            self.testcase.pk,
+            {
+                'default_tester': self.new_author.pk,
+            }
+        )
+
+        self.testcase.refresh_from_db()
+
+        self.assertEqual(self.new_author.pk, self.testcase.default_tester.pk)
+
+    def test_update_default_tester_accepts_username(self):
+        self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+            self.testcase.pk,
+            {
+                'default_tester': self.new_author.username,
+            }
+        )
+
+        self.testcase.refresh_from_db()
+
+        self.assertEqual(self.new_author.pk, self.testcase.default_tester.pk)
+
+    def test_update_default_tester_accepts_email(self):
+        self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+            self.testcase.pk,
+            {
+                'default_tester': self.new_author.email,
+            }
+        )
+
+        self.testcase.refresh_from_db()
+
+        self.assertEqual(self.new_author.pk, self.testcase.default_tester.pk)
```

Comment:
Also needs tests for negative scenarios, when id/username/email are not something from the DB.